seshdash/views.py

```diff
@@ -41,16 +44,47 @@ def index(request,site_id=0):
         site_id = sites[0].pk
     #Check if user has any sites under their permission
     context_dict, content_json = get_user_data(request.user,site_id,sites)
+
     context_dict = jsonify_dict(context_dict,content_json)
     #Generate date for dashboard  TODO use victron solar yield data using mock weather data for now
     y_data,x_data = prep_time_series(context_dict['site_weather'],'cloud_cover','date')
     y2_data,x2_data = prep_time_series(context_dict['site_weather'],'cloud_cover','date')
     #create graphs for PV dailt prod vs cloud cover
-    context_dict =  linebar(x2_data,y_data,y2_data,'WH produced','% cloud cover','chartcontainer',context_dict)
-    print "len x1: %s, x2 len: %s len y1:%s, len y2: %s"%(len(x_data),len(x2_data),len(y_data),len(y2_data))
-    #pprint( context_dict)
-    return render(request,'seshdash/main-dash.html',context_dict)
+    data='me'
+
+    context_dict=linebar(x2_data,y_data,y2_data,'WH produced','% cloud cover','chartcontainer',context_dict)
+    context_dict['test'] = 'test'
+
+    now2 = datetime.datetime.now()
```

Comment:
I would suggest moving all this code to a seperate function like create_weather_chart_Data or something like that in order to keep this function shorter. Else it will get too complicated.